includes/admin/load.php

```diff
@@ -224,31 +228,85 @@ function perflab_enqueue_modules_page_scripts() {
 	wp_enqueue_script( 'plugin-install' );
 
 	wp_enqueue_script(
-		'perflab-plugin-management',
-		plugin_dir_url( __FILE__ ) . 'js/perflab-plugin-management.js',
+		'perflab-install-activate-plugins',
+		plugin_dir_url( __FILE__ ) . 'js/perflab-install-activate-plugins.js',
 		array( 'jquery' ),
```

Comment:
Do you think this can be omitted now? I don't see jQuery being used with the script.

Reply:
Thank you for bringing this up. Following other feedback, I've opted for the admin action approach, so I've removed this JavaScript.